store/migrate_test.go

```diff
@@ -185,6 +230,13 @@ type ACIInfoV0_2 struct {
 	Latest     bool
 }
 
+type ACIInfoV3 struct {
+	BlobKey    string
+	Name       string
+	ImportTime time.Time
+	Latest     bool
+}
+
```

Comment:
In answer to your question: this is no longer outstanding, as we now retain the original ordering